controller/RestTestCenterUsers.php

```diff
@@ -191,6 +223,8 @@ public function delete()
             $this->returnJson([
                 'success' => $this->getService()->unassignUser($testCenter, $user, $role)
             ]);
+        } catch (\common_exception_NotFound $e) {
+            return $this->returnFailure($e);
         } catch (\common_Exception $e) {
```

Comment:
Consider change to `catch (\Exception $e)` or `catch (\Throwable $e)`

Reply:
Done, thanks